### PR TITLE
Done

### PR DESCRIPTION
**Pull request by anniezh12**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,6 +23,19 @@
...
+          <div class="view-profile-div">
+            <h3 class="view-profile-text">View Profile</h3>
+          </div>
+          <div class="card-text-container">
+            <h4 class="student-name">Alexis Roman </h4>
+            <p class="student-location">New York, NY</p>
+          </div>
+        </a>
+      </div>
+
```
Comment (bhollan) [inline index.html]: Why are you adding more than one student card?  And why above the `roster-cards-container` tag?

**Comment by bhollan** [comment]
Hi @anniezh12 !

Looking through your changes here, everything seems ok except it seems like you've added 2 cards to `index.html`.  If you can fix that and commit the changes, we should be able to merge this in.  Let us know if you need any help!

**Comment by anniezh12** [comment]
Hi Brian ,
Sorry If I made that mistake I might pasted it twice let me see and will let you know
Aniqa

-----Original Message-----

**Comment by anniezh12** [comment]
Hi Brian ,
I don't understand what 2 cards are you mentioning I only see mine once can you double check and please guide me or can you fix it?
Aniqa

-----Original Message-----

**Comment by ivalentine** [comment]
Hi @anniezh12!

There appears to be a stray card on lines 27–37 of `index.html`, outside of the `roster-cards-container` `div` where those cards belong. It looks like that card also exists inside of that `div`, so you can just delete the one on lines 27–37, and you should be good to go.

If you need further clarification on this change, just let us know here. If you need help making the change, feel free to Ask a Question on Learn.

Thanks!

cc @aturkewi

**Comment by bhollan** [comment]
Hi @anniezh12 !

I think you have overwritten a fellow student's card in `index.html`.  To see what we're seeing, just click on the "Files Changed" tab towards the top of the pull request on github and it will highlight on the left, the way the file USED to look, and on the right, how it looks in your suggested changes.  I don't think you did it intentionally, I think it's just how things ended up from all the intermittent changes since starting this lab.

As of now, you need to re-instate Brian Reynold's card in `index.html`.  You can Ask A Question about this lab if you need help getting it to work out.  That's what we're here for!

CC: @aturkewi

**Comment by bhollan** [comment]
Hi @anniezh12 !

I'm going to close this PR for now.  We can re-open it any time you like.  Just let us know whenever you're ready to implement the necessary changes.

If you need any help, as always, please feel free to Ask A Question!

CC: @aturkewi

**Pull request closed without merging**